From 8be8a53c339ce76a9bc517c85ec6e34064ffe9f4 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Fri, 13 May 2022 12:04:53 +0300
Subject: [PATCH 12/30] arm: dts: s32c: Add SPI properties from Linux

Make use of the properties used in Linux Kernel dtb for
SPI nodes, except the dma which doesn't make sense in U-Boot.

Issue: ALB-8823
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/dts/s32cc.dtsi | 26 +++++++++++++++++++++++++-
 1 file changed, 25 insertions(+), 1 deletion(-)

diff --git a/arch/arm/dts/s32cc.dtsi b/arch/arm/dts/s32cc.dtsi
index 4a132a3f19..ae2b257bc7 100644
--- a/arch/arm/dts/s32cc.dtsi
+++ b/arch/arm/dts/s32cc.dtsi
@@ -216,6 +216,10 @@
 			clocks = <&clks S32GEN1_SCMI_CLK_SPI_MODULE>;
 			clock-names = "dspi";
 			spi-num-chipselects = <8>;
+			bus-num = <0>;
+			spi-fifo-size = <5>;
+			spi-cpol;
+			spi-cpha;
 			// For U-Boot 2020.04 only
 			num-cs = <8>;
 			#address-cells = <1>;
@@ -229,10 +233,14 @@
 			clocks = <&clks S32GEN1_SCMI_CLK_SPI_MODULE>;
 			clock-names = "dspi";
 			spi-num-chipselects = <5>;
+			bus-num = <1>;
+			spi-fifo-size = <5>;
+			spi-cpol;
+			spi-cpha;
+			// For U-Boot 2020.04 only
 			num-cs = <5>;
 			#address-cells = <1>;
 			#size-cells = <0>;
-			// For U-Boot 2020.04 only
 			status = "disabled";
 		};
 
@@ -242,6 +250,10 @@
 			clocks = <&clks S32GEN1_SCMI_CLK_SPI_MODULE>;
 			clock-names = "dspi";
 			spi-num-chipselects = <5>;
+			bus-num = <2>;
+			spi-fifo-size = <5>;
+			spi-cpol;
+			spi-cpha;
 			// For U-Boot 2020.04 only
 			num-cs = <5>;
 			#address-cells = <1>;
@@ -312,6 +324,10 @@
 			clocks = <&clks S32GEN1_SCMI_CLK_SPI_MODULE>;
 			clock-names = "dspi";
 			spi-num-chipselects = <5>;
+			bus-num = <3>;
+			spi-fifo-size = <5>;
+			spi-cpol;
+			spi-cpha;
 			// For U-Boot 2020.04 only
 			num-cs = <5>;
 			#address-cells = <1>;
@@ -325,6 +341,10 @@
 			clocks = <&clks S32GEN1_SCMI_CLK_SPI_MODULE>;
 			clock-names = "dspi";
 			spi-num-chipselects = <5>;
+			bus-num = <4>;
+			spi-fifo-size = <5>;
+			spi-cpol;
+			spi-cpha;
 			// For U-Boot 2020.04 only
 			num-cs = <5>;
 			#address-cells = <1>;
@@ -338,6 +358,10 @@
 			clocks = <&clks S32GEN1_SCMI_CLK_SPI_MODULE>;
 			clock-names = "dspi";
 			spi-num-chipselects = <5>;
+			bus-num = <5>;
+			spi-fifo-size = <5>;
+			spi-cpol;
+			spi-cpha;
 			// For U-Boot 2020.04 only
 			num-cs = <5>;
 			#address-cells = <1>;
-- 
2.17.1

